fcrepo-serialization/src/main/java/org/fcrepo/serialization/BaseFedoraObjectSerializer.java

```diff
@@ -19,35 +19,33 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 
+import javax.inject.Inject;
 import javax.jcr.Repository;
-import javax.jcr.RepositoryException;
 import javax.jcr.Session;
 
 import org.fcrepo.kernel.api.models.FedoraResource;
 import org.fcrepo.kernel.api.exception.InvalidChecksumException;
-import org.springframework.beans.factory.annotation.Autowired;
 
 /**
  * Auto-wire some helpful services for the FedoraObjectSerializer
  *
  * @author cbeer
  */
-public abstract class BaseFedoraObjectSerializer implements
-        FedoraObjectSerializer {
+public abstract class BaseFedoraObjectSerializer implements FedoraObjectSerializer {
```

Comment:
@awoods: all of this will go away. hang on for another commit: your review will be much easier.